apps/emails/templates/nrh-default-contribution-confirmation-email.html

```diff
@@ -4,329 +4,250 @@
-              <div style="border-collapse: collapse;display: table;width: 100%;background-color: transparent;">
-                <!--[if (mso)|(IE)]><table width="100%" cellpadding="0" cellspacing="0" border="0"><tr><td style="padding: 0px;background-color: #ffffff;" align="center"><table cellpadding="0" cellspacing="0" border="0" style="width:550px;"><tr style="background-color: #ffffff;"><![endif]-->
-
-                <!--[if (mso)|(IE)]><td align="center" width="550" style="background-color: #ffffff;width: 550px;padding: 0px;border-top: 0px solid transparent;border-left: 0px solid transparent;border-right: 0px solid transparent;border-bottom: 0px solid transparent;" valign="top"><![endif]-->
-                <div class="u-col u-col-100"
-                  style="max-width: 320px;min-width: 550px;display: table-cell;vertical-align: top;">
-                  <div style="background-color: #ffffff;width: 100% !important;">
-                    <!--[if (!mso)&(!IE)]><!-->
+          <div class="u-row-container">
+            <table role="presentation" cellpadding="0" cellspacing="0" width="100%" border="0">
+              <tbody>
+                <tr>
+                  <td
+                    style="overflow-wrap:break-word;word-break:break-word;padding:10px 10px 15px;"
+                    align="left">
                     <div
-                      style="padding: 0px;border-top: 0px solid transparent;border-left: 0px solid transparent;border-right: 0px solid transparent;border-bottom: 0px solid transparent;">
-                      <!--<![endif]-->
-
-                      <table style="font-family:'Lato',sans-serif;" role="presentation" cellpadding="0" cellspacing="0"
-                        width="100%" border="0">
-                        <tbody>
-                          <tr>
-                            <td
-                              style="overflow-wrap:break-word;word-break:break-word;padding:10px 10px 15px;font-family:'Lato',sans-serif;"
-                              align="left">
-
-                              <div
-                                style="color: #7e8c8d; line-height: 140%; text-align: center; word-wrap: break-word;">
-                                <p style="font-size: 14px; line-height: 140%;">@{{ copyright_year }} {{ org_name }} All
-                                  rights reserved.</p>
-                              </div>
-
-                            </td>
-                          </tr>
-                        </tbody>
-                      </table>
-
-                      <!--[if (!mso)&(!IE)]><!-->
+                      style="color: #7e8c8d; line-height: 140%; text-align: center; word-wrap: break-word;">
+                      <p style="font-size: 14px; line-height: 140%; font-family: Roboto, sans-serif;">©{{ copyright_year }} {{ org_name }} All
```

Comment:
This should also reference `rp_name`, not `org_name`.

```suggestion
                      <p style="font-size: 14px; line-height: 140%; font-family: Roboto, sans-serif;">©{{ copyright_year }} {{ rp_name }} All
```